cmd/sensuctl/main_windows.go

```diff
@@ -0,0 +1,26 @@
+package main
```

Comment:
Could you add a comment similar to what we find in `cmd/sensu-agent/main_windows.go` that basically explains why we have this file (that's essentially a copy of `main.go`)? Took me a bit to understand that 😅